demoextendsymfonyform/composer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "prestashop/demoextendsymfonyform-2",
```

Comment:
I think it's better to modify the folder name too.
`demoextendsymfonyform/` => `demoextendsymfonyform-2`

Reply:
I think the naming `demoextendsymfonyform-2` is not the best for module as it becomes not found in the marketplace if I change is folder name to demoextendsymfonyform-2 and cannot do the same for the class name.